src/SerialFirmata.cpp

```diff
@@ -66,6 +66,11 @@ boolean SerialFirmata::handleSysex(byte command, byte argc, byte *argv)
                 // because all Arduino pins are set to OUTPUT by default in StandardFirmata.
                 pinMode(pins.rx, INPUT);
               }
+#if defined(SERIAL_STORE_AND_FORWARD)    
```

Comment:
Define this block at the top of the SERIAL_CONFIG case. That way you only need to declare it once, since the only variable it depends on is `portId`.